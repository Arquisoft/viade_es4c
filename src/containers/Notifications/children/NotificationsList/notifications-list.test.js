import React from 'react';
import { shallow } from 'enzyme';
import NotificationList from "./notifications-list.component";

describe('NotificationList', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<NotificationList debug />);
  
    expect(component).toMatchSnapshot();
  });
});
