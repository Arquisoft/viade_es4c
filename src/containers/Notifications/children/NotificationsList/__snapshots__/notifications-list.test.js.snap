// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationList should render correctly in "debug" mode 1`] = `<div />`;
